decidim-admin/app/views/decidim/admin/attachments/_form.html.erb

```diff
@@ -23,7 +23,7 @@
     </div>
 
     <div class="row column">
-      <%= form.upload :file, optional: false %>
+      <%= form.upload :file, optional: false, redesigned: redesign_enabled? %>
```

Comment:
Do we need to pass this in every `form.upload` call separately or couldn't you check this at the form builder itself?

Within the `FormBuilder`, you will have access to `@template` which is the view. Through that method you can call the methods available in the view.

Reply:
You are right, I think it will work fine, I've changed this to upload method internals